mara_pipelines/commands/files.py

```diff
@@ -177,3 +178,51 @@ def html_doc_items(self) -> List[Tuple[str, str]]:
                  _.tt[json.dumps(self.null_value_string) if self.null_value_string is not None else None]),
                 ('time zone', _.tt[self.timezone]),
                 (_.i['shell command'], html.highlight_syntax(self.shell_command(), 'bash'))]
+
+
+class WriteFile(sql._SQLCommand):
+    """Writes data to a local file. The command is executed on the shell."""
+
+    def __init__(self, dest_file_name: str, sql_statement: Optional[Union[Callable, str]] = None, sql_file_name: Optional[str] = None,
+                 replace: Optional[Dict[str, str]] = None, db_alias: Optional[str] = None,
+                 compression: Compression = Compression.NONE,
+                 format: formats.Format = formats.CsvFormat()) -> None:
+        """
+        Writes the output of a sql query to a file in a specific format. The command is executed on the shell.
```

Comment:
`... on the shell on the machine which runs mara (e.g. in the docker container!).`

Reply:
That is true! I am not happy with that everything in Mara runs through a shell command. For example, when I run mara in a Jupyter Notebook, it is pretty stupid that I have to carry out commands to make sure that the shell toolings are installed. It would be much smarter to use sqlalchemy or a DB API instead. I had already in mind to add a `SqlExecutionContext` which doesn't use shell but the DB API or SQLAlchemy, but ... didn't really had a usecase where I needed it...